Misc/NEWS.d/next/Documentation/2020-02-23-16-40-51.bpo-39710.GU_1Bb.rst

```diff
@@ -0,0 +1 @@
+"will be returned as unicode" reminiscent from Python 2
```

Comment:
```suggestion
Removed from the calendar documentation "will be returned as unicode" wording reminiscent from Python 2
```

Reply:
I think we can skip news on this change.